src/main/java/io/tackle/pathfinder/services/AssessmentSvc.java

```diff
@@ -12,14 +20,15 @@
 import javax.validation.constraints.NotNull;
 import javax.ws.rs.BadRequestException;
 
+import java.util.ArrayList;
 import java.util.List;
 import java.util.Optional;
 
 @ApplicationScoped
 @Log
 public class AssessmentSvc {
     @Inject
-    AssessmentMapper mapper;
+    public AssessmentMapper mapper;
```

Comment:
Just for my understanding, why did you add  `public`? any special reason?

Reply:
To be honest, I've been experiencing few times that Quarkus was complaining about the injection of this AssessmentMapper, and apparently it got solved by making it public.

Reply:
But it has to be another thing because leaving it as package now it works.... I experience some random issues with CDI and Quarkus....